airflow/www/templates/airflow/dags.html

```diff
@@ -554,5 +552,33 @@ <h2>DAGs</h2>
       $(".loading-task-stats").remove();
       $(".loading-dag-stats").remove();
     }
+
+    function showSvgTooltip(text, offsetTop, offsetLeft) {
+      var tip = $('#svg-tooltip');
+      tip.children('.tooltip-inner').text(text);
+      var centeringOffset = tip.width() / 2;
+      tip.css({
+        "display": "block",
+        "left": offsetLeft + 12.5 - centeringOffset + 'px',// 12.5 == half of circle width
```

Comment:
Rather than hard-coding these offsets, would it be cleaner/less future error prone to instead pass in the result of `getBoundingClientRect`, i.e. this function beceomes:

```js
     function showSvgTooltip(text, elemClientRect) { 
```

or 
```js
     function showSvgTooltip(text, elemTop, elemLeft, elemBottom, elemRight ) { 
```

(TLBR, instead of TLXY as IEs' don't have X and Y attributes according to https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect)

Reply:
I simplified the function argument similar to your first suggestion.

Your IE comment caused some confusion for me—I was thinking that I had used the unsupported `getboundingclientrect` properties, though that wasn't the fact. I did end up confirming that this works in IE Edge. I also tested in an IE11 instance, but the UI failed to operate properly. Those are either much larger compatibility issues or due to unresponsiveness of the VirtualBox instance I was testing in.

Reply:
Whops sorry!